index.html

```diff
@@ -225,23 +227,25 @@ <h2>Extensions to the <dfn data-cite=
       <section>
         <h2><dfn>getEntries()</dfn> method</h2>
         <p>Returns a <a>PerformanceEntryList</a> object returned by <a href=
-        "#filter-buffer-by-name-and-type"></a> algorithm with <var>buffer</var>
-        set to <a>relevant performance entry buffer</a>, and <var>name</var>
-        and <var>type</var> set to <code>null</code>.</p>
+        "#filter-buffers-by-name-and-type"></a> algorithm with <var>buffers</var>
+        set to <a>relevant performance entry buffer</a> for every type,
+        and <var>name</var> and <var>type</var> set to <code>null</code>.</p>
       </section>
       <section>
         <h2><dfn>getEntriesByType()</dfn> method</h2>
         <p>Returns a <a>PerformanceEntryList</a> object returned by <a href=
-        "#filter-buffer-by-name-and-type"></a> algorithm with <var>buffer</var>
-        set to <a>relevant performance entry buffer</a>, <var>name</var> set to
-        <code>null</code>, and <var>type</var> set to `type`.</p>
+        "#filter-buffers-by-name-and-type"></a> algorithm with <var>buffer</var>
```

Comment:
Should it be 'buffers' here too, and below?

Reply:
getEntriesByType is only ever one type, so one buffer.  getEntriesByName should indicate buffers since we are looking in potentially all buffers.

Reply:
We spoke offline and this is now updated.